src/lib.rs

```diff
@@ -390,8 +390,17 @@ fn find_vcpkg_target(cfg: &Config, target_triplet: &TargetTriplet) -> Result<Vcp
     let vcpkg_root = try!(find_vcpkg_root(&cfg));
     try!(validate_vcpkg_root(&vcpkg_root));
 
-    let mut base = vcpkg_root.clone();
-    base.push("installed");
+    let cargo_base_path =
+        PathBuf::from(env::var("CARGO_MANIFEST_DIR").expect("we use cargo, this exists"));
+    let is_manifest_mode = env::var_os("VCPKGRS_IGNORE_MANIFEST_MODE").is_none()
```

Comment:
Thanks for doing this.

Thinking more about this, I think perhaps this is the wrong approach. Exposing another environment variable here to disable manifest mode in the presence of the json file seems overly complicating to add to the exposed interface, feels like a "you aren't going to need it" sort of feature.

I think perhaps a more useful approach would be to defer back to your original change without the environment variable override, but we add some sanity checks at the end here to verify that the target installed directory exists?  That way, we can give a useful error message depending on whether manifest mode or not was used (telling the user to run either `cargo vcpkg install` for non-manifest mode, and `vcpkg(.exe) install` when manifest mode is detected), and otherwise we just make it silently work.

What do you think?